aviary/interface/reports.py

```diff
@@ -42,3 +52,118 @@ def subsystem_report(prob, **kwargs):
 
     for subsystem in core_subsystems.values():
         subsystem.report(prob, reports_folder, **kwargs)
+
+
+def mission_report(prob, **kwargs):
+    """
+    Creates a basic mission summary report that is place in the "reports" folder
+
+    Parameters
+    ----------
+    prob : AviaryProblem
+        The AviaryProblem used to generate this report
+    """
+    def _get_phase_value(traj, phase, var_name, units, indices=None):
+        try:
+            vals = prob.get_val(f"{traj}.{phase}.timeseries.{var_name}",
+                                units=units,
+                                indices=indices,)
+
+        except KeyError:
+            try:
+                vals = prob.get_val(f"{traj}.{phase}.states:{var_name}",
+                                    units=units,
+                                    indices=indices,)
+            except KeyError:
+                try:
+                    vals = prob.get_val(f"{traj}.{phase}.timeseries.states:{var_name}",
+                                        units=units,
+                                        indices=indices,)
+                except KeyError:
+                    try:
+                        vals = prob.get_val(f"{traj}.{phase}.{var_name}",
+                                            units=units,
+                                            indices=indices,)
+                    # 2DOF breguet range cruise uses time integration to track mass
+                    except TypeError:
+                        vals = prob.get_val(f"{traj}.{phase}.timeseries.time",
+                                            units=units,
+                                            indices=indices,)
+                    except KeyError:
+                        vals = None
```

Comment:
could this be changed to something like:
```
key_paths = [f"timeseries.{var_name}", f"states:{var_name}", f"timeseries.states:{var_name}", f"{var_name}", "timeseries.time"]
vals = None
for key_path in key_paths:
	try:
		vals = prob.get_val(f"{traj}.{phase}.{key_path}", units=units, indices=indices,)
		return vals
	except KeyError:
		pass
```

Reply:
That will work for everything except catching the TypeError that only shows up for the timeseries.time check. I'll play around a little with this idea